src/vs/workbench/contrib/terminal/browser/xterm/xtermTerminal.ts

```diff
@@ -442,6 +440,7 @@ export class XtermTerminal extends Disposable implements IXtermTerminal, IDetach
 	}
 
 	resize(columns: number, rows: number): void {
+		this._logService.debug(`Resizing terminal to ${columns} col, ${rows} row`);
```

Comment:
We can save making a bunch of unique strings in the JS engine that will need to be garbage collected by logging like this:

```suggestion
		this._logService.debug('resizing', columns, rows);
```